HLTrigger/Configuration/python/HLT_75e33_cff.py

```diff
@@ -237,6 +237,8 @@
 
 fragment.load('L1Trigger.Configuration.GTemulator_cff')
 fragment.l1tGTProducer.GMTTkMuons = cms.InputTag("l1tTkMuonsGmt")
+#fragment.l1tGTProducer.CL2Jets = cms.InputTag("l1tSCPFL1PuppiCorrectedEmulator")
+#fragment.l1tGTProducer.CL2HtSum = cms.InputTag("l1tSCPFL1PuppiCorrectedEmulatorMHT")
```

Comment:
Not sure if the HLT workflows will run with the recent change of the L1 input collections.
To be removed if the workflows run.